desktop/cmp/input/ButtonGroupInput.js

```diff
@@ -65,23 +72,26 @@ export class ButtonGroupInput extends HoistInput {
             const active = (this.renderValue === value);
             return React.cloneElement(button, {
                 active,
-                // key is a workaround for https://github.com/palantir/blueprint/issues/3971
-                key: `${active} ${value}`,
+                intent,
                 minimal: withDefault(minimal, false),
+                outlined: withDefault(outlined, false),
                 disabled: withDefault(btnDisabled, false),
                 onClick: () => {
                     if (enableClear) {
                         this.noteValueChange(active ? null : value);
                     } else {
                         this.noteValueChange(value);
                     }
-                }
+                },
+                // Workaround for https://github.com/palantir/blueprint/issues/3971
+                key: `${active} ${value}`
             });
         });
 
         return buttonGroup({
             items: buttons,
             ...buttonGroupProps,
+            minimal: withDefault(minimal, outlined, false),
```

Comment:
What's going on here? Do 'outlined' buttons need to be in a 'minimal' buttonGroup?

Reply:
Yes, it's a great question. 

BP supports the minimal modifier on ButtonGroup but not outlined. Prior to this change, you could set `outlined:true` on the _child_ buttons and it would work, but you could not set it on the group itself. Seemed nice to make it more consistent since we're in the business of unpacking these props anyway and applying them to children ourselves.

BUt I need to look again though at what this specific default is doing - if anything - it might not be required.